src/command_line.py

```diff
@@ -9,12 +9,14 @@
 
 __parser.add_argument('--concurrency-per-core', '-c', type=int, default=2, dest='concurrency',
                       help='Number of workers = concurrency * CPU_CORES_COUNT')
-__parser.add_argument('--temp-dir', '-t', type=str, default='./tmp',
-                      help='Base directory to place all downloaded files before bundling')
-__parser.add_argument('--output-dir', '-o', type=str, default='./out',
-                      help='Base directory to place bundled zip file in')
 __parser.add_argument('--input-dir', '-i', type=str, default='./in',
                       help='Base directory where list files are located')
+__parser.add_argument('--output-dir', '-o', type=str, default='./out',
+                      help='Base directory to place bundled zip file in')
+__parser.add_argument('--packager', '-p', type=str, default='m',
+                      help='Which packager to use (m for memory based packager or f for FS based packager)')
+__parser.add_argument('--temp-dir', '-t', type=str, default='./tmp',
+                      help='Base directory to place all downloaded files before bundling. Applies only for FS based packager')
```

Comment:
as discussed previously, I would prefer to stick to each os temp folder convention.

`/tmp` for Linux 
`%TEMP%` for windows

Reply:
Done